test/tests/unit/measurement/schema/schema.test.ts

```diff
@@ -1129,24 +1144,23 @@ describe('command schema', async () => {
 		it('should pass (tld)', () => {
 			const input = {
 				type: 'dns',
-				target: '.com',
+				target: 'com',
```

Comment:
OP of an issue used ".com" as a value for tld. Shouldn't that be supported as well? @MartinKolarik

Reply:
I know, but that was probably a typo. It's either `com` or `com.` in the spec and all the existing tools (try `dig`).